docs/index.md

```diff
@@ -374,7 +382,20 @@ spec:
     - w-spcloud123456
 ```
 
-This plugin will automatically create a table called `kubernetes_certificate`:
+### Set custom_resource_tables parameter in config file
+
+You may specify the custom_resource_tables with singular name, full name or wild card values:
+
+- custom_resource_tables = ["certificate","certificates.cert-manager.io","certificates.*"]
+
+```hcl
+connection "kubernetes" {
+  plugin     = "kubernetes"
+  custom_resource_tables = ["certificate"]
+}
+```
+
+Based on the above config file setup this plugin will automatically create a table called `kubernetes_certificate`:
```

Comment:
```suggestion
Steampipe will create table schemas for all custom resource definitions set in the `custom_resource_tables` argument.

For instance, if my connection configuration is:

```hcl
connection "kubernetes" {
  plugin = "kubernetes"

  custom_resource_tables = ["certificates.*"]
}
```

Steampile will automatically create the `kuberentes_certificate` table, which can then be inspected and queried like other tables:

```bash
.inspect kubernetes_certificate;
+---------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| column                    | type                     | description                                                                                                 |
+---------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
| _ctx                      | jsonb                    | Steampipe context in JSON form, e.g. connection_name.                                                       |
| additional_output_formats | jsonb                    | AdditionalOutputFormats defines extra output formats of the private key and signed certificate chain to be  |
|                           |                          | written to this Certificate's target Secret. This is an Alpha Feature and is only enabled with the `--featu |
|                           |                          | re-gates=AdditionalCertificateOutputFormats=true` option on both the controller and webhook components.     |
| api_version               | text                     | The API version of the resource.                                                                            |
| common_name               | text                     | CommonName is a common name to be used on the Certificate. The CommonName should have a length of 64 charac |
|                           |                          | ters or fewer to avoid generating invalid CSRs. This value is ignored by TLS clients when any subject alt n |
|                           |                          | ame is set. This is x509 behaviour: https://tools.ietf.org/html/rfc6125#section-6.4.4                       |
| conditions                | jsonb                    | List of status conditions to indicate the status of certificates. Known condition types are `Ready` and `Is |
|                           |                          | suing`.                                                                                                     |
| creation_timestamp        | timestamp with time zone | CreationTimestamp is a timestamp representing the server time when this object was created.                 |
| dns_names                 | jsonb                    | DNSNames is a list of DNS subjectAltNames to be set on the Certificate.                                     |
| duration                  | text                     | The requested 'duration' (i.e. lifetime) of the Certificate. This option may be ignored/overridden by some  |
|                           |                          | issuer types. If unset this defaults to 90 days. Certificate will be renewed either 2/3 through its duratio |
|                           |                          |n or `renewBefore` period before its expiry, whichever is later. Minimum accepted duration is 1 hour. Value |
|                           |                          |  must be in units accepted by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration              |
| email_addresses           | jsonb                    | EmailAddresses is a list of email subjectAltNames to be set on the Certificate.                             |
| encode_usages_in_request  | boolean                  | EncodeUsagesInRequest controls whether key usages should be present in the CertificateRequest               |
| ip_addresses              | jsonb                    | IPAddresses is a list of IP address subjectAltNames to be set on the Certificate.                           |
| is_ca                     | boolean                  | IsCA will mark this Certificate as valid for certificate signing. This will automatically add the `cert sig |
|                           |                          | n` usage to the list of `usages`.                                                                           |
| issuer_ref                | jsonb                    | IssuerRef is a reference to the issuer for this certificate. If the `kind` field is not set, or set to `Iss |
|                           |                          | uer`, an Issuer resource with the given name in the same namespace as the Certificate will be used. If the  |
|                           |                          | `kind` field is set to `ClusterIssuer`, a ClusterIssuer with the provided name will be used. The `name` fie |
|                           |                          | ld in this stanza is required at all times.                                                                 |
| keystores                 | jsonb                    | Keystores configures additional keystore output formats stored in the `secretName` Secret resource.         |
| kind                      | text                     | Type of resource.                                                                                           |
| labels                    | jsonb                    | Map of string keys and values that can be used to organize and categorize (scope and select) objects. May m |
|                           |                          | atch selectors of replication controllers and services.                                                     |
| last_failure_time         | text                     | LastFailureTime is the time as recorded by the Certificate controller of the most recent failure to complet |
|                           |                          | e a CertificateRequest for this Certificate resource. If set, cert-manager will not re-request another Cert |
|                           |                          | ificate until 1 hour has elapsed from this time.                                                            |
| literal_subject           | text                     | LiteralSubject is an LDAP formatted string that represents the [X.509 Subject field](https://datatracker.ie |
|                           |                          | tf.org/doc/html/rfc5280#section-4.1.2.6). Use this *instead* of the Subject field if you need to ensure the |
|                           |                          |  correct ordering of the RDN sequence, such as when issuing certs for LDAP authentication. See https://gith |
|                           |                          | ub.com/cert-manager/cert-manager/issues/3203, https://github.com/cert-manager/cert-manager/issues/4424. Thi |
|                           |                          | s field is alpha level and is only supported by cert-manager installations where LiteralCertificateSubject  |
|                           |                          | feature gate is enabled on both cert-manager controller and webhook.                                        |
| name                      | text                     | Name of resource.                                                                                           |
| namespace                 | text                     | Namespace defines the space within which each name must be unique.                                          |
| not_after                 | text                     | The expiration time of the certificate stored in the secret named by this resource in `spec.secretName`.    |
| not_before                | text                     | The time after which the certificate stored in the secret named by this resource in spec.secretName is vali |
|                           |                          | d.                                                                                                          |
| private_key               | jsonb                    | Options to control private keys used for the Certificate.                                                   |
| renew_before              | text                     | How long before the currently issued certificate's expiry cert-manager should renew the certificate. The de |
|                           |                          | fault is 2/3 of the issued certificate's duration. Minimum accepted value is 5 minutes. Value must be in un |
|                           |                          | its accepted by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration                            |
| renewal_time              | text                     | RenewalTime is the time at which the certificate will be next renewed. If not set, no upcoming renewal is s |
|                           |                          | cheduled.                                                                                                   |
| revision_history_limit    | bigint                   | revisionHistoryLimit is the maximum number of CertificateRequest revisions that are maintained in the Certi |
|                           |                          | ficate's history. Each revision represents a single `CertificateRequest` created by this Certificate, eithe |
|                           |                          | r when it was created, renewed, or Spec was changed. Revisions will be removed by oldest first if the numbe |
|                           |                          | r of revisions exceeds this number. If set, revisionHistoryLimit must be a value of `1` or greater. If unse |
|                           |                          | t (`nil`), revisions will not be garbage collected. Default value is `nil`.                                 |
| secret_name               | text                     | SecretName is the name of the secret resource that will be automatically created and managed by this Certif |
|                           |                          | icate resource. It will be populated with a private key and certificate, signed by the denoted issuer.      |
| secret_template           | jsonb                    | SecretTemplate defines annotations and labels to be copied to the Certificate's Secret. Labels and annotati |
|                           |                          | ons on the Secret will be changed as they appear on the SecretTemplate when added or removed. SecretTemplat |
|                           |                          | e annotations are added in conjunction with, and cannot overwrite, the base set of annotations cert-manager |
|                           |                          |  sets on the Certificate's Secret.                                                                          |
| uid                       | text                     | UID is the unique in time and space value for this object.                                                  |
| uris                      | jsonb                    | URIs is a list of URI subjectAltNames to be set on the Certificate.                                         |
| usages                    | jsonb                    | Usages is the set of x509 usages that are requested for the certificate. Defaults to `digital signature` an |
|                           |                          | d `key encipherment` if not specified.                                                                      |
+---------------------------+--------------------------+-------------------------------------------------------------------------------------------------------------+
```

```bash
> select name, uid, kind, api_version, namespace from kubernetes_certificate;
+------------------------------------+--------------------------------------+-------------+--------------------+-----------+
| name                               | uid                                  | kind        | api_version        | namespace |
+------------------------------------+--------------------------------------+-------------+--------------------+-----------+
| temporal-w-spcloudt6t6sk7toegg-tls | 5ccd69be-6e73-4edc-8c1d-bccd6a1e6e38 | Certificate | cert-manager.io/v1 | default   |
+------------------------------------+--------------------------------------+-------------+--------------------+-----------+
```
```

Reply:
Updated as suggested.